src/info/info_redis/info_redis.c

```diff
@@ -199,6 +210,7 @@ void AddToInfo_Indexes(RedisModuleInfoCtx *ctx, TotalIndexesInfo *total_info) {
   RedisModule_InfoAddFieldULongLong(ctx, "number_of_active_indexes_indexing", total_info->num_active_indexes_indexing);
   RedisModule_InfoAddFieldULongLong(ctx, "total_active_write_threads", total_info->total_active_write_threads);
   RedisModule_InfoAddFieldDouble(ctx, "total_indexing_time", (float)total_info->indexing_time / (float)CLOCKS_PER_MILLISEC);
+  RedisModule_InfoAddFieldULongLong(ctx, "total_documents_indexed", total_info->total_documents_indexed);
```

Comment:
I think we should name `total_documents_indexed_by_*_field` and `total_documents_indexed` in a way that emphasizes that they reflect different values to avoid confusion (the first is an always growing counter, while the second reflects a temporary state)
i suggest:
`total_documents_indexed_*_field` (removed `by`)
`total_indexes_num_docs`

WDYT

Reply:
sounds good